client/js/stores/snippet-store.js

```diff
@@ -154,6 +166,32 @@ class SnippetStore {
       }
     }
   }
+
+  destroySnippet (data) {
+    if (data.ok){
+      var oldSnippets = this.state.snippets;
+      var oldUserSnippets = this.state.currentUserSnippets;
+
+      oldSnippets.forEach(function (snippet) {
```

Comment:
maybe `.filter` suits better for this than '.forEach' ?
https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Obiekty/Array/filter